spec/models/post_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) { User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'New user', posts_counter: 1) }
+  subject do
+    Post.new(author: user, title: 'Nature', text: 'This is exciting!', likes_counter: 0, comments_counter: 0)
+  end
+
+  before { subject.save }
+
+  it 'should be valid' do
+    expect(subject).to be_valid
+  end
+
+  it 'should have comments_counter equal to 0' do
+    expect(subject.comments_counter).to eq(0)
+  end
+
+  it 'should have likes_counter equal to 0' do
+    expect(subject.likes_counter).to eq(0)
+  end
+
+  it 'title should not be greater than 250 characters' do
+    post = Post.create(author: user, title: 'a' * 251, text: 'This is exciting!')
+    expect(post).not_to be_valid
+  end
+
+  it 'return last five posts' do
+    expect(subject.display_comments).to eq(subject.comments.order(updated_at: :desc).limit(5))
+  end
+
+  it 'increments the author\'s posts_counter after save' do
+    expect { subject.save }.to change { user.reload.posts_counter }.by(1)
+  end
+end
```

Comment:
Please note that some of your tests are not passing 😢. Kindly fix this.
![Screenshot from 2023-07-15 19-10-10](https://github.com/mercymugambi/blog-app/assets/44624138/03fd170e-bbf6-4341-b63f-f5d5158c684c)

Reply:
![Capture](https://github.com/mercymugambi/blog-app/assets/48310207/5feeccd6-c96f-4526-96d6-5819770bb89e)